WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -660,7 +660,7 @@ ZmPreviewView.prototype.set = function(item) {
 
         //Send everything trough ConvertD
```

Comment:
We can remove this comment, it's misleading
@dasiyogesh any idea if onlyoffice formatter is part of convertd or separate one?

Reply:
its separate

Reply:
removed comment.